readthedocs/projects/tasks.py

```diff
@@ -184,6 +185,7 @@ def run(self, pk, version_pk=None, build_pk=None, record=True, docker=False,
 
         if self.build_env.failed:
             self.send_notifications()
+        build_complete.send(sender=Build, build=self.build_env.build)
```

Comment:
On second thought, `build_env_complete` doesn't really make sense, because there are multiple buildenvs, and only one build -- the build is what is important here. We can always implement `build_env` signals separately, not sure it does much good to pass in the `build_env` to a signal describing a build object.